src/app/pages/send/send-crypto-asset-form/components/recipient-accounts-dialog/recipient-accounts-dialog.tsx

```diff
@@ -23,27 +23,31 @@ export function RecipientAccountsDialog() {
 
   if (stacksAddressesNum === 0 && btcAddressesNum === 0) return null;
   const accountNum = stacksAddressesNum || btcAddressesNum;
-  const maxAccountsShown = accountNum > 10 ? 10 : accountNum;
 
   return (
-    <Dialog header={<Header variant="dialog" title="My accounts" />} isShowing onClose={onGoBack}>
-      <Virtuoso
-        height={virtuosoHeight}
-        style={{
-          ...virtuosoStyles,
-          height: `calc(${virtuosoHeight * maxAccountsShown}px + ${getHeightOffset(true, true)}px)`,
-        }}
-        itemContent={index => (
-          <Box key={index} my="space.05" px="space.05">
-            <AccountListItem
-              stacksAccount={stacksAccounts[index]}
-              onClose={onGoBack}
-              index={index}
-            />
-          </Box>
-        )}
-        totalCount={stacksAddressesNum || btcAddressesNum}
-      />
+    <Dialog
+      header={<Header variant="dialog" title="My accounts" />}
+      isShowing
+      onClose={onGoBack}
+      wrapChildren={false}
+    >
+      <VirtuosoWrapper>
```

Comment:
`Virtuoso` height is now set to `100%` and the actual height is controlled by a wrapping container